airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/assets/dev_assets.py

```diff
@@ -0,0 +1,166 @@
+import yaml
+from dagster import Output, asset
+
+"""
+NOTE: This file is temporary and will be removed once we have metadata files checked into source control.
+
+TODO (ben): Remove this file once we have metadata files checked into source control.
```

Comment:
this file is meant to be short lived as once weve created these metadata files and checked them into source control we no long need to generate them